README.md

```diff
@@ -12,15 +12,23 @@ Demo calls are a great chance to find out about tools and initiatives (both old
 
 Demo calls happen (usually) on the last Thursday of the month at 7pm GMT.
 
-* First call: Thursday 1st February 2018, 7pm GMT.
-  * [Watch again on YouTube](https://www.youtube.com/watch?v=Qo-bFqMUbLo) and check out our supplementary [video with Guiomar](https://www.youtube.com/watch?v=VyTPIuzqdU4) which we recorded after some technical challenges during the main call.
-* Second call: [Thursday 22nd February 2018, 7pm GMT](https://www.timeanddate.com/worldclock/converter.html?iso=20180222T190000&p1=tz_gmt).
+* March call: [Thursday 22nd March 2018, 7pm GMT](https://www.youtube.com/watch?v=vrX5IZH3aIM)
```

Comment:
Should we put the time zone converter here instead of the youtube link? Or something like 

Our next call will be [live on youtube](https://www.youtube.com/watch?v=vrX5IZH3aIM) on [Thursday 22nd March 2018, 7pm GMT](https://www.timeanddate.com/worldclock/converter.html?iso=20180322T190000&p1=tz_gmt)!

Reply:
I like that second option. I kinda like the youtube link becuase it gives you this countdown when you click on it, and you can subscribe to the channel etc. But lets go with your one because it does both!